docs/dev/app/listings.md

```diff
@@ -1,9 +1,390 @@
 # Listing Pages
 
-Basic structure; feel free to deviate.
+All three listing pages contain a filter bar at the top, a scrollable list of Listing Cards on the left, and the Listing Details of the currently selected listing card on the right. Differences in display between listing pages are listed below. Then, the usage and functionality of components used to bulid the listing pages are explained. 
 
-## Cases
+## Listing Page Comparison 
 
-## Limited Case Assignments
+| Listing Page | Case | Limited Case Assignments | Language Support |
+|--------------|------|---------------------|------------------------------|
+| Filters | 1. Remote/In Person <br/> 2. Languages  <br/> 3. Adjudicating Agency <br/> 4. Country of Origin | 1. Country Field <br/> 2. Language(s) | 1. Listing Type <br/> 2. Language(s) |
+| Listing Highlights | 1. Relief sought <br/> 2. Time Commitment <br/> 3. Remote/In Person <br/> 4. Adjudicating Agency <br/> 5. Client Languages <br/> 6. Client Country of Origin <br/> 7. Client Location* | 1. Country Field <br/> 2. Language(s) <br/> 3. Expected Deliverable | **DOC** <br/> 1. Language(s) <br/> 2. Number of Pages  <br/>  <br/> **CASE_INT** <br/> 1. Language(s)  <br/> 2. Time Commitment  <br/> 3. Remote/In Person  <br/>  <br/> **INT**  <br/> 1. Language(s) <br/> 2. Remote/In Person |
 
-## Language Support
+::: note (*) Note: 
+If the client is in custody (i.e., `is_detained`, pulled from LegalServer, is `true`), "Client Location" appears as "Custody Location." Otherwise, it remains as Client Location.
+::: 
+
+## **Components Used**
+
+- `ListingPage`
+    - `ListingCard` 
+    - `ListingDetails`
+        - `ProfileMatch`
+        - `InterestForm`
+
+## `ListingPage`
+
+::: note Description
```

Comment:
do we like the note or the tip better? 
<img width="769" alt="Screen Shot 2024-05-31 at 1 05 36 AM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/7e032ab9-5849-4b84-a7ea-ea69a5cb5232">
<img width="769" alt="Screen Shot 2024-05-31 at 1 05 26 AM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/b1cbf97f-7509-4637-9459-0fc17fe57843">

Reply:
i think the top one looks better since it gives the block a bit more contrast

Reply:
gotcha, changed!